react-intl/index.d.ts

```diff
@@ -34,14 +34,16 @@ declare namespace ReactIntl {
         formats?: any;
     }
 
-    interface InjectedIntl {
+    interface InjectedIntlProps {
+      intl: {
```

Comment:
I know that it isn't working and thats why you can probably see **[WIP]** within the title, can't you?